Addition of full file paths in security documentation (#8113)
opensearch-project/documentation-website: _security/configuration/security-admin.md

## A word of caution

> If you make changes to the configuration files in `config/opensearch-security`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index. The `securityadmin.sh` file can typically be found at `/tools/securityadmin.sh`

Reviewer: For consistency-sake, can we make all paths relative to `<OPENSEARCH_HOME>` so this path would be `<OPENSEARCH_HOME>/plugins/opensearch-security/tools/securityadmin.[sh|bat]`

> If you make changes to the configuration files in `config/opensearch-security`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index. The `securityadmin.sh` file can typically be found at `<OPENSEARCH_HOME>/plugins/opensearch-security/tools/securityadmin.[sh|bat]`

Reviewer 2: Let's avoid using the word "typically" unless we can call out why the file would be in a different location.

```suggestion
If you make changes to the configuration files in `config/opensearch-security`, OpenSearch does _not_ automatically apply these changes. Instead, you must run `securityadmin.sh` to load the updated files into the index. The `securityadmin.sh` file can be found in `<OPENSEARCH_HOME>/plugins/opensearch-security/tools/securityadmin.[sh|bat]`.
```

Running `securityadmin.sh` **overwrites** one or more portions of the `.opendistro_security` index. Run it with extreme care to avoid losing your existing resources. Consider the following example:

1. You initialize the `.opendistro_security` index.
1. You create ten users using the REST API.

> 1. You decide to create a new [reserved user]({{site.url}}{{site.baseurl}}/security/access-control/api/#reserved-and-hidden-resources) using `internal_users.yml` (Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/internal_users.yml`)

Reviewer 2: ```suggestion
1. You decide to create a new [reserved user]({{site.url}}{{site.baseurl}}/security/access-control/api/#reserved-and-hidden-resources) using `internal_users.yml`, found in `<OPENSEARCH_HOME>/config/opensearch-security/` directory.
```
